inference-engine/src/gna_plugin/frontend/layer_quantizer.hpp

```diff
@@ -642,7 +642,11 @@ class DataQuantizer<Desc, InferenceEngine::WeightableLayer *> : public DataQuant
  public:
     explicit DataQuantizer(float scaleFactor) : DataQuantizerBase(scaleFactor) {}
     bool operator()(InferenceEngine::WeightableLayer *wl) const {
-        (*this)(wl, typename Desc::MandatoryType());
+        if (LayerInfo(wl).isConcatAlignFilter()) {
```

Comment:
Will this work also for networks without FakeQuantize layers? I think this solution should be only for cases with FakeQuantizes when we get precision from their levels parameter. Otherwise they should be quantized using Mandatory type.